queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -0,0 +1,202 @@
+        filing: Filing,
+        filing_date_time: str,
+        effective_date: str) -> list:
+    # pylint: disable=too-many-locals, too-many-branches, too-many-statements, too-many-arguments
+    """Get the outputs for the Continuation In notification."""
+    pdfs = []
+    attach_order = 1
+    headers = {
+        'Accept': 'application/pdf',
+        'Authorization': f'Bearer {token}'
+    }
+
+    if status == Filing.Status.PAID.value:
+        # add filing pdf
+        filing_pdf = requests.get(
+            f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}',
+            headers=headers
+        )
+        if filing_pdf.status_code != HTTPStatus.OK:
+            logger.error('Failed to get pdf for filing: %s', filing.id)
+        else:
+            filing_pdf_encoded = base64.b64encode(filing_pdf.content)
+            pdfs.append(
+                {
+                    'fileName': 'Continuation Application.pdf',
+                    'fileBytes': filing_pdf_encoded.decode('utf-8'),
+                    'fileUrl': '',
+                    'attachOrder': str(attach_order)
+                }
+            )
+            attach_order += 1
+
+        corp_name = business.get('legalName')
+        receipt = requests.post(
+            f'{current_app.config.get("PAY_API_URL")}/{filing.payment_token}/receipts',
+            json={
+                'corpName': corp_name,
+                'filingDateTime': filing_date_time,
+                'effectiveDateTime': effective_date if effective_date != filing_date_time else '',
+                'filingIdentifier': str(filing.id),
+                'businessNumber': business.get('taxId', '')
+            },
+            headers=headers
+        )
+        if receipt.status_code != HTTPStatus.CREATED:
+            logger.error('Failed to get receipt pdf for filing: %s', filing.id)
+        else:
+            receipt_encoded = base64.b64encode(receipt.content)
+            pdfs.append(
+                {
+                    'fileName': 'Receipt.pdf',
+                    'fileBytes': receipt_encoded.decode('utf-8'),
+                    'fileUrl': '',
+                    'attachOrder': str(attach_order)
+                }
+            )
+            attach_order += 1
+    elif status == Filing.Status.COMPLETED.value:
+        # add certificate of continuation
+        certificate = requests.get(
+            f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}'
+            '?type=certificateOfContinuation',
+            headers=headers
+        )
+        if certificate.status_code != HTTPStatus.OK:
+            logger.error('Failed to get certificate of continuation for filing: %s', filing.id)
+        else:
+            certificate_encoded = base64.b64encode(certificate.content)
+            pdfs.append(
+                {
+                    'fileName': 'Certificate of Continuation.pdf',
```

Comment:
Can you confirm the name is `Certificate of Continuation` in UI design its just `Certificate`
![image](https://github.com/bcgov/lear/assets/64869009/66391230-2cf4-4c89-9469-98ba18db0d69)

Reply:
Referred to [Email spreadsheet](https://docs.google.com/spreadsheets/d/17wMY9znmWMr2RoXH9S2elffC8uE7iifUGB5Hu3VVFpE/edit?gid=0#gid=0) 
I will check.

Reply:
Andy confirmed it should be 'Certificate of Continuation'. He also updated the designs.

Reply:
@ketaki-deodhar can you update here https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/core/meta/filing.py#L273 from `certificate` to `certificateOfContinuation`